tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,48 @@
+"""Sandwich calculator."""
+import collections
+import pyinputplus as pyip
+
+BREAD_COST = {"wheat": 1, "white": 1.5, "sourdough": 1.5}
+MEAT_COST = {"chicken": 3, "turkey": 4, "ham": 3, "tofu": 4}
+CHEESE_COST = {"cheddar": 2, "swiss": 1.5, "mozzarella": 1}
+SAUCE_COST = {"mayo": 0.2, "mustard": 0.1, "lettuce": 0.2, "tomato": 0.1}
+
+
+def print_section_header(string: str):
+    """Print title section."""
+    print(f' {string} '.center(20, '='))
+
+
+def make_sandwiches():
+    """Build sandwich."""
+    options = collections.defaultdict(int)
+    print("Welcome to the sandwich ordering program!!! "
+          "Please choose ingredients for your sandwich:")
+    print_section_header('BREAD')
+    bread = pyip.inputMenu(list(BREAD_COST.keys()))
+    options[bread] = 1
```

Comment:
I think here you can set the `options[bread] = bread_price` and it will simplify the logic down the pipeline